### DISCUSSION TITLE
Add option to manage Suppressions manually

### DISCUSSION THREAD

**Discussion by dragoangel** [postalserver/postal#1496] [Feature suggestions]
In case of bots spam attacks etc to quickly fix situation and have time to investigate your webapp you need somehow stop mail from sending to evil recipients and still be delivered to good customers. This can be done with suppressions that Postal already have, but there no way to remove email from suppresion or to add email to suppresion list. As well good to have option temporary suppress for example one domain: *@evil.com or allow suppression by regular expression for advanced users.

**Comment by grrinch** [comment]
+1
